hangupsbot/plugins/_chatbridge/chatbridge_syncrooms.py

```diff
@@ -59,25 +59,55 @@ def _send_to_external_chat(self, config, event):
         message = event.passthru["original_request"]["message"]
         image_id = event.passthru["original_request"]["image_id"]
 
+        is_action = event.passthru["chatbridge"].get("source_action")
+
         if not message:
             message = ""
 
+        if any(a.type == 4 for a in event.conv_event._event.chat_message.annotation):
+            # This is a /me message sent from desktop Hangouts.
+            is_action = True
+            # The user's first name prefixes the message, so try to strip that.
```

Comment:
Indeed, it's purely a client-side thing (possibly only on desktop too, I've never seen it recognised on Android).  If you start a message with `/me`, it's replaced with what Hangouts deduces to be your first name, and the annotation flag is set.